Minutes/Readme.md

```diff
@@ -0,0 +1,44 @@
+# W3C WoT CG Meeting Minutes
+
+This folder contains the minutes taken during the W3C WoT CG meetings.
+
+An initial save of the minutes take place and are committed to save the unedited state.
+A single html file is saved from the W3C server without the additional CSS and JS that comes in a folder.
+Since we are not planning to host the final versions, it does not make sense to save them in a pleasant-to-look-at version.
+
+A manual or automatic post processing takes place to do the following:
+
+- Removing mention of draft: 
+
+```html
+<h1>WoT CG at TPAC 2022</h1>
+```
+
+becomes 
+
+```html
+<h1>WoT CG at TPAC 2022</h1>
```

Comment:
The two strings are the same. Maybe you changed also the readme?

Reply:
Oh replace all effected it as well 😅

Reply:
The original state is at https://github.com/w3c/wot-cg/pull/11/commits/57533669b32810d0e5757db9d70583fb8d08c5ff  . I will change this